libs/csm/rest/csm_rest_quota.py

```diff
@@ -62,15 +66,20 @@ def get_user_quota(self, uid, quota_type, **kwargs):
         self.log.info("Get user quota request successfully sent...")
         return response
 
-    def iam_user_quota_payload(self):
+    def iam_user_quota_payload(self): #pylint disable=no-self-use
```

Comment:
Can u move the comment to top of function

Reply:
done